add Bonita Cloud backup request (#1551)
bonitasoft/bonita-doc: modules/ROOT/pages/define-and-deploy-the-bdm.adoc

> You are recommended xref:back-up-bonita-bpm-platform.adoc[Back up your platform] and database before deploying a new BDM in case of problems during the update.

Reviewer: ```suggestion
It is recommended to xref:back-up-bonita-bpm-platform.adoc[Back up the platform] and database before deploying a new BDM in case of problems during the update.
```

====

[NOTE.bonitacloud,caption=Bonita Cloud]
====

> In Bonita Cloud, a dedicated backup of the database can be done upon request. Just open a support ticket with a time frame and we'll sync with you.

Reviewer: ```suggestion
In Bonita Cloud, a dedicated backup of the database can be done upon request. Just open a support ticket with a time frame and we'll synchronize with you.
```
